fix(docs): correct examples to use company.team namespace (#1495)
kestra-io/docs: content/docs/09.administrator-guide/03.monitoring.md

          - FAILED
          - WARNING
      - type: io.kestra.plugin.core.condition.ExecutionNamespaceCondition
        namespace: company.analytics
        prefix: true
```

Adding this single flow will ensure that you receive a Slack alert on any flow failure in the `company.analytics` namespace. Here is an example alert notification:

![alert notification](/docs/administrator-guide/alert-notification.png)

::alert{type="warning"}
Note that if you want this alert to be sent on failure across multiple namespaces, you will need to add an ``OrCondition`` to the ``conditions`` list. See the example below:
```yaml
id: alert

> namespace: company.monitoring

Reviewer: ```suggestion
namespace: company.system
```

      - type: io.kestra.plugin.core.condition.OrCondition
        conditions:
          - type: io.kestra.plugin.core.condition.ExecutionNamespaceCondition

>             namespace: company.analytics

Reviewer: Product team?

```suggestion
            namespace: company.product
```

            prefix: true
          - type: io.kestra.plugin.core.condition.ExecutionFlowCondition
            flowId: cleanup

>             namespace: company.monitoring

Reviewer: I'd keep this as a system namespace. We want to encourage people to have a system namespace for these type of flows, but it should fall under the company namespace.

```suggestion
            namespace: company.system
```

          - FAILED
          - WARNING
      - type: io.kestra.plugin.core.condition.ExecutionNamespaceCondition

>         namespace: company.analytics

Reviewer: ```suggestion
        namespace: company.product
```

        prefix: true
      - type: io.kestra.plugin.core.condition.ExecutionFlowCondition
        flowId: cleanup

>         namespace: company.monitoring

Reviewer: ```suggestion
        namespace: company.system
```

```

> Here, there's no overlap between the two conditions. The first condition will only match executions in the `company.analytics` namespace, while the second condition will only match executions from the `cleanup` flow in the `company.monitoring` namespace. If you want to match executions from the `cleanup` flow in the `company.monitoring` namespace **or** any execution in the `product` namespace, make sure to add the `OrCondition`.

Reviewer: ```suggestion
Here, there's no overlap between the two conditions. The first condition will only match executions in the `company.product` namespace, while the second condition will only match executions from the `cleanup` flow in the `company.system` namespace. If you want to match executions from the `cleanup` flow in the `company.system` namespace **or** any execution in the `product` namespace, make sure to add the `OrCondition`.
```
